runtime/parachains/src/configuration.rs

```diff
@@ -43,6 +43,8 @@ pub struct HostConfiguration<BlockNumber> {
 	pub max_code_size: u32,
 	/// The maximum head-data size, in bytes.
 	pub max_head_data_size: u32,
+	/// THe maximum POV block size, in bytes.
+	pub max_pov_size: u32,
```

Comment:
Will we need a runtime upgrade to initialize this to a non-0 value? If yes, what should we use as a default? I haven't really played with those yet.

Reply:
I think that just setting it in the three genesis locations will do it for us. Note: as of now, this is only set for rococo and the test function. If it should also be set for polkadot/kusama, then more changes are needed.

Reply:
No runtime migration needed!